data-prepper-core/src/test/java/com/amazon/dataprepper/peerforwarder/PeerForwarderAppConfigTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.peerforwarder;
+
+import com.amazon.dataprepper.parser.model.DataPrepperConfiguration;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+class PeerForwarderAppConfigTest {
+
+    private static final PeerForwarderAppConfig peerForwarderAppConfig = new PeerForwarderAppConfig();
```

Comment:
We should test this using Spring. This will help verify that the code works with Spring. We can either: 1) Change this to an integration test, or 2) make a separate test (`PeerForwarderAppConfigIT`); or 3) Make a full integration suite for PeerForwarder which does use Spring.

Here is an example from the plugin model. The tests are integration tests for the whole plugin framework (similar to the option 3 above), so you don't necessarily want to copy all of that. But, you can see how it uses Spring to load an app config.

https://github.com/opensearch-project/data-prepper/blob/31bf29b5c562ed74cecd65760c38c0a91edbca00/data-prepper-core/src/test/java/com/amazon/dataprepper/plugin/DefaultPluginFactoryIT.java#L40-L51